applications/nrf5340_audio/README.rst

```diff
@@ -0,0 +1,922 @@
+|                       | Solid yellow - External hardware codec selected.                                                    |
+|                       | This LED turns solid yellow also when the devices are reset, for the time then pins are floating.   |
++-----------------------+-----------------------------------------------------------------------------------------------------+
+| **FTDI SPI**          | Off - No data is written to the hardware codec using SPI.                                           |
+|                       +-----------------------------------------------------------------------------------------------------+
+|                       | Yellow - The same SPI is used for both the hardware codec and the SD card.                          |
+|                       | When this LED is yellow, the shared SPI is used by the FTDI to write data to the hardware codec.    |
++-----------------------+-----------------------------------------------------------------------------------------------------+
+| **IFMCU**             | Off - No PC connection available.                                                                   |
+| (bottom side)         +-----------------------------------------------------------------------------------------------------+
+|                       | Solid green - Connected to PC.                                                                      |
+|                       +-----------------------------------------------------------------------------------------------------+
+|                       | Rapid green flash - USB enumeration failed.                                                         |
++-----------------------+-----------------------------------------------------------------------------------------------------+
+| **HUB**               | Off - No PC connection available.                                                                   |
+| (bottom side)         +-----------------------------------------------------------------------------------------------------+
+|                       | Green - Standard USB hub operation.                                                                 |
++-----------------------+-----------------------------------------------------------------------------------------------------+
+
+.. _octave_configuration:
+
+Configuration
+*************
+
+|config|
+
+.. _octave_configuration_repos:
+
+Setting up the nRF5340 Audio repositories
+=========================================
+
+The application relies on the following :ref:`external OSS repositories <dm_code_base>` that need to be pulled using west:
+
+* LC3 software codec repository
+* SBC software codec repository
+* Hardware codec driver repository
+
+To have these repositories managed by west, complete the following steps:
+
+1. Add the LC3 codec repository to the west manifest file of your project.
```

Comment:
This adds only LC3 codec repository or all 3 repositories?

Reply:
If it adds only LC3 coded, then how to add other repositiories ?

Reply:
Addressed in a separate PR (to source branch, rebase pending).